### ISSUE TITLE
App crashes when trying to configure

### ISSUE THREAD

**Issue by fa7ad** [guyzmo/git-repo#158]
I updated to the latest version today by doing a `pip3 install --upgrade`

I removed the gitlab, bitbucket and github config from by .gitconfig

And then I tried to run `git-repo config`

It crashes with this output:

![screenshot from 2017-05-20 13-14-39](https://cloud.githubusercontent.com/assets/7447677/26273944/5df542fe-3d5e-11e7-9b7b-f8a453fc40b8.png)

**Comment by guyzmo** [comment]
Hello, @fa7ad !

could you try again with `git-repo config -v` to pinpoint where exactly it has crashed and copy the stacktrace over here?

**Comment by guyzmo** [comment]
The bitbucket support is quite new, and the token creation is done using some webscraping magic, so it's possible that with the latest UI changes on bitbucket.org something broke 😉

**Comment by fa7ad** [comment]
> could you try again with git-repo config -v to pinpoint where exactly it has crashed and copy the stacktrace over here?

Sure thing, Here you go
```log
Do you want to configure the bitbucket service?
    [Yn]> 
Is your service self-hosted?
    [yN]> 
Please enter your credentials to connect to the service:
username> evilinfinite7@gmail.com
password> 
» Login to bitbucket…
» Generating app password…
» App password is setup with following scopes:
»     account, team, project:write, repository:admin, repository:delete
»     pullrequest:write, snippet, snippet:write
Great! You've been identified 🍻
Do you want to give a custom name for this service's remote?
    [yN]> n
Do you want to configure a git alias?
N.B.: instead of typing `git repo bb` you'll be able to type `git bb`
    [Yn]> 
Fatal error: decoding to str: need a bytes-like object, tuple found
------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/git_repo/repo.py", line 583, in main
    return GitRepoRunner(args).run()
  File "/usr/lib/python3.6/site-packages/git_repo/kwargparse.py", line 68, in run
    return self._action_dict[frozenset(args)](self)
  File "/usr/lib/python3.6/site-packages/git_repo/repo.py", line 575, in do_config
    setup_service(service)
  File "/usr/lib/python3.6/site-packages/git_repo/repo.py", line 562, in setup_service
    service.store_config(self.config, **new_conf)
  File "/usr/lib/python3.6/site-packages/git_repo/services/service.py", line 126, in store_config
    config.set_value(section, option, value)
  File "/usr/lib/python3.6/site-packages/git/config.py", line 79, in assure_data_present
    return func(self, *args, **kwargs)
  File "/usr/lib/python3.6/site-packages/git/config.py", line 90, in flush_changes
    rval = non_const_func(self, *args, **kwargs)
  File "/usr/lib/python3.6/site-packages/git/config.py", line 571, in set_value
    self.set(section, option, self._value_to_string(value))
  File "/usr/lib/python3.6/site-packages/git/config.py", line 554, in _value_to_string
    return force_text(value)
  File "/usr/lib/python3.6/site-packages/gitdb/utils/encoding.py", line 29, in force_text
    return text_type(data, encoding)
TypeError: decoding to str: need a bytes-like object, tuple found
```

**Comment by guyzmo** [comment]
ok that bug should be solved in latest release: 10.0.3. Sorry for the inconvenience ☺

**Comment by fa7ad** [comment]
Thanks for the quick update :smile:
